configs/stage/roles/inventory.json

```diff
@@ -5,6 +5,7 @@
       "description": "Perform any available operation against any Inventory resource.",
       "system": true,
       "platform_default": true,
+      "admin_default": true,
```

Comment:
as this role has also `platform_default` set to true - it basically gives all permissions to your app for everyone.
Preferred way is to have more granularity on your permissions. If possible it would be great to have more permissions instead of `"inventory:*:*"` but also have read or write actions or also to have also for resources for format of permissions:

`<application>:<resource>:<action>.` 

or if this is not doable - 
 it would be enough to have also `platform_default` set to false.

Let me know whether is it is feasible for you or if you need more info.
thanks !

Reply:
For now, Inventory has only one resource type - "hosts". With the coming "Inventory Groups" there will probably be more in the future, but not for now.

The reason why I want to set `true` for both `platform_default` and `admin_default` is mentioned in the PR description. By default, we want all users to be able to do anything in their inventory - read, create, update, delete... hosts, that's why `platform_default` was always set to `true` here. The problem is that when an org admin decides to delete this permission from the default user group, then the admin himself also loses the permissions for interacting with the Inventory. And that's why I would like to set both `platform_default` and `admin_default` to `true`.

Reply:
I am sorry for big delay!

I was discussing this also with security team and when we give admin access by default with wildcard permission - it is not good security practice.

it is best to define more granular permission that way what `resources` and `actions` are intentional.
is this possible to do it ? 

More info here:https://docs.google.com/document/d/1xfpASXhi6C87dl7r1IVSt3a64yrSBqgMgp8YtVvtTa4/edit

thanks!

Reply:
yes it looks great! thank you for doing those changes!